src/scanner/scan-parameter-generator.ts

```diff
@@ -20,11 +20,11 @@ export class ScanParameterGenerator {
         };
 
         if (options == null || options.testsToRun == null) {
-            result.runOnly!.values = Object.keys(this.rulesIncluded).filter(
-                id => this.rulesIncluded[id].status === 'included',
-            );
+            result.runOnly!.values = Object.keys(this.rulesIncluded)
+                .filter(id => this.rulesIncluded[id].status === 'included')
+                .concat('frame-tested');
```

Comment:
It feels off to split the logic of "which rules get included" between the existing place (`get-rule-inclusions`) and this hard-coded magic rule. I think it would be preferable to keep the knowledge of which rules are included in one place.

I'd recommend adding a new `status` value (maybe `included-always` or `force-included` or something) along with a comment indicating the difference between "included" (in automated checks by default) vs "included-always" (even when running scans of an otherwise limited rule set). Then the reason for always-including `frame-tested` can be documented in `get-rule-inclusions` in its `reason` field, like we already do for all the other cases where we override axe rule enablement.